### DISCUSSION TITLE
Import map diagnostics: - Invalid top-level key "hash". Only "imports" and "scopes" can be present.

### DISCUSSION THREAD

**Discussion by borsemayur2** [crewdevio/Trex#90] [Q&A] [answered]
Getting mentioned message for following import_map.json file?
```
{
  "imports": {
    "denodb": "https://deno.land/x/denodb/mod.ts",
    "oak": "https://deno.land/x/oak/mod.ts",
    "oak_graphql": "https://deno.land/x/oak_graphql/mod.ts"
  },
  "hash": {
    "denodb": "c69c2c8a891f8db67af6c03e2e6e8738e2889949a0d8f415fa5b1b22dac8d02b",
    "oak_graphql": "a4d70645b3798dbe2574868d8640ec42679290e77f308331a0b9de6036779c3f",
    "oak": "26e45e79380e5ba551993537a73526ae81a55c63e981a318c5c5e47aa4fbd8c3"
  }
}
```

Original issue reference from `velociraptor`:
https://github.com/jurassiscripts/velociraptor/issues/56

**Comment by buttercubz** [comment, marked as answer]
Thanks for the report, apparently Deno now inform about the structure that the import maps should have, for now it only launches the warning but I will be working on a way to solve the problem without losing the functionality

**Reply by borsemayur2** [reply]
Okay. Thanks for the reply.
